Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1798,6 +1873,12 @@ public async ValueTask DisposeAsync()
                 _idleTimer = null;
             }
 
+            if(_inactiveTimer is not null)
```

Comment:
This should fix the build failure.

```suggestion
            if (_inactiveTimer is not null)
```

Reply:
Thanks did make the change locally, but didn't push the change.